server/routes/checkAnswers.js

```diff
@@ -21,7 +22,109 @@ module.exports = function Index({ formService, authenticationMiddleware }) {
     '/:bookingId',
     asyncMiddleware(async (req, res) => {
       const errors = req.flash('errors')
-      res.render('pages/check-answers', { data: res.locals.formObject, bookingId: req.params.bookingId, errors })
+      const { bookingId } = req.params
+
+      // offender details
+      const offenderDetail = await offenderService.getOffenderDetails(res.locals.user.token, bookingId)
+
+      // retrieve form inputs
+      // eslint-disable-next-line
+      const loadForm = async req => {
+        const { form_response: formObject = {}, id: formId } = await formService.getFormResponse(
+          req.user.username,
+          bookingId
+        )
+        return { formId, formObject }
+      }
+      const { formObject } = await loadForm(req, res)
```

Comment:
you could merge this method and the call:
 
```
      const { form_response: formObject = {} } = await formService.getFormResponse(
          req.user.username,
          bookingId
        )
```

Reply:
ok